raster/src/main/scala/geotrellis/raster/Tile.scala

```diff
@@ -292,4 +292,10 @@ trait Tile extends CellGrid with IterableTile with MappableTile[Tile] {
     }
     s
   }
+
+  def isNoDataTile = {
+    var empty = true
+    foreach { z => if (isData(z)) empty = false }
```

Comment:
+@lossyrob I think the name is appropriate. I remember wanting a function like this but stopped just short of committing it.